apps/spruce/src/components/Header/Navbar.tsx

```diff
@@ -75,13 +79,24 @@ export const Navbar: React.FC = () => {
         >
           <StyledAnimatedIcon icon={HolidayTree} />
         </LogoLink>
-        <PrimaryLink
-          data-cy="project-health-link"
-          onClick={() => sendEvent({ name: "Clicked project health link" })}
-          to={getCommitsRoute(projectIdentifier)}
-        >
-          Project Health
-        </PrimaryLink>
+        {spruceWaterfallEnabled ? (
```

Comment:
Can we add the inverse link to the navbar Auxiliary Dropdown? i.e. project health if a user is in the beta and waterfall if not.

Reply:
good idea! added!
i additionally gated it on the `showWaterfallPage` condition b/c I don't think we should show the links if we haven't released the project yet... lmk if that doesnt make sense lol